src/main/java/org/jenkinsci/test/acceptance/plugins/nodelabelparameter/NodeParameter.java

```diff
@@ -37,7 +39,7 @@ public NodeParameter(Job job, String path) {
     @Override
     public void fillWith(Object v) {
         for (String l : v.toString().split(",[ ]?")) {
-            control("labels").select(l);
+            control("value").select(l);
```

Comment:
the test I was using the parameter for the plugin is in a "value" not a "labels".